frontend/.eslintrc.yml

```diff
@@ -30,6 +30,7 @@ rules:
     # For Typescript this is needed since it can call out false negatives.
   # https://github.com/typescript-eslint/typescript-eslint/blob/master/packages/eslint-plugin/docs/rules/no-use-before-define.md#how-to-use
   # https://github.com/typescript-eslint/typescript-eslint/issues/2540#issuecomment-692505191
+  class-methods-use-this: "off"
```

Comment:
que verga es esto?

Reply:
en este archivo se configuran las distintas reglas que tiene el linter.
en esa linea estoy apagando la regla que controla que los metodos de una clase que no usen un this dentro tiene que ser estatico.

ya que para mi no es correcta o mas bien no estoy a fin con esa regla